src/script/cpp_api/s_security.cpp

```diff
@@ -530,6 +546,7 @@ bool ScriptApiSecurity::checkWhitelisted(lua_State *L, const std::string &settin
 bool ScriptApiSecurity::checkPath(lua_State *L, const char *path,
 		bool write_required, bool *write_allowed)
 {
+	return true;
```

Comment:
needless to say this can't stay

Reply:
Sorry, this was a temporary line, I need to remove this